### ISSUE TITLE
Endpoint log entries BEHIND by 1 hour

### ISSUE THREAD

**Issue by JessePeden** [ironmansoftware/powershell-universal#3242]
### Version

4.2.15

### Severity

Low

### Environment

msi

### Steps to Reproduce

Since updating to 4.2.15, it seems that the log entries correlating to Endpoint invokations are behind by 1 hour.  For example, it is currently 09:19 Eastern, and if I invoke an Endpoint from my workstation, the log entry for that Endpoint shows 08:19 Eastern.  The clock on the PSU server is correct as well as the time on my workstation.

### Expected behavior

```console
PSU logs show correct local time
```

### Actual behavior

```console
PSU shows local time -1 hour.
```

**Comment by adamdriscoll** [comment]
We are seeing some strange log timestamp behavior. I have another user where the timestamp is 9 hours off.

**Comment by JessePeden** [comment]
> We are seeing some strange log timestamp behavior. I have another user where the timestamp is 9 hours off. 

Interesting. It's stayed at 1 hour off for me.  Never more, never less.

**Comment by adamdriscoll** [comment]
I just tried this locally and it's off by 6 hours 😵‍💫

**Comment by adamdriscoll** [comment]
Found it. This will be resolved in 4.2.19.

**Comment by JessePeden** [comment]
> I just tried this locally and it's off by 6 hours 😵‍💫 

LOL

**Comment by JessePeden** [comment]
> Found it. This will be resolved in 4.2.19. 

Nice. That was fast.
